charts/openshift-metering/templates/presto/_presto_helpers.tpl

```diff
@@ -45,6 +45,35 @@ connector.name=blackhole
 connector.name=memory
 {{ end }}
 
+{{- define "presto-prometheus-catalog-properties" -}}
```

Comment:
Should this be wrapped in a if condition, e.g.:
```yaml
{{- if .Values.presto.spec.config.connectors.prometheus.enabled }}
{{- define ... }}
```

Reply:
yeah, seems like otherwise it will be there doing nothing when not enabled. Done in ce5e2ad

Reply:
Hmm, but with a change like that, helm template is puking on the location of the define now...

Reply:
Swapped order of define/if, helm template seems good with that. Done in 6315e14